### ISSUE TITLE
Не правильное определение стульев в комнате на дату

### ISSUE THREAD

**Issue by DmitriyLuckyman** [ArtemIsaakian/Sigma_Tz#1]
https://github.com/ArtemIsaakian/Sigma_Tz/blob/b4547b2a5d1e3e7cb5c605d896c757d1f5c90a1d/function.php#L233

В этом запросе мы получаем список стульев, которые были перемешены  в комнута в определённую дату
А нам необходимо получить список стульев в этой комнате на дату. Это разные выборки.

**Comment by ArtemIsaakian** [comment]
Правильно ли я понимаю, что пользователь должен выбрать дату, чтобы узнать,
в каких комнатах находятся стулья на выбранную им дату? Или пользователь
должен выбрать дату и комнату чтобы узнать какие стулья в комнате на
выбранную дату находятся?

Чт, 2 сент. 2021 г. в 17:31, DmitriyLuckyman ***@***.***>:

**Comment by DmitriyLuckyman** [comment]
Чтобы было прощё давайте сделаем распределение стульев по комнатам на дату.

**Comment by ArtemIsaakian** [comment]
Правильно ли я понимаю, что Вы имеете в виду следующее:
Пользователь выбирает дату, потом стулья и комнаты, в которые их надо
переместить и нажимает на кнопку «распределить». В итоге должна получиться
таблица со стульями и комнатами, в которых они находятся на выбранную дату?

Пт, 3 сент. 2021 г. в 12:06, DmitriyLuckyman ***@***.***>:

**Comment by DmitriyLuckyman** [comment]
Нет. Не надо ничего распределять. Необходимо увидеть состояние на дату
